composer.json

```diff
@@ -55,5 +55,7 @@
             "dev-master": "3.0-dev",
             "2.0": "2.0-dev"
         }
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
Because of [datasource](https://github.com/fsi-open/admin-bundle/blob/master/composer.json#L35) dev dependency in `fsi/admin-bundle`. Composer will refuse to resolve the dependency set because by default, `minimum-stability` is `stable`.